notifications_utils/recipients.py

```diff
@@ -276,6 +276,25 @@ def rows_with_missing_data(self):
     def rows_with_message_too_long(self):
         return self._filter_rows("message_too_long")
 
+    @property
+    def rows_with_combined_variable_content_too_long(self):
```

Comment:
where is this function being called?

Reply:
[In admin where we check rows for errors and build error messages to display on the page.](https://github.com/cds-snc/notification-admin/pull/1765/files#diff-ac7b8d56e509c921efaadb5a776e1c9037531c4d2af78787f06f67a1f3533ae4R212 )